api/test/robot_tests/plugin_manager/plugin_manager.robot

```diff
@@ -105,6 +106,7 @@ POST /plugins-manager/{id}/execute already started
     ...    json=${data}
 
     ${expectedError}=    Set Variable
+        # TODO: thyra-plugin-hello-world to be renamed to station-massa-hello-world when we have new release on hello-world plugin
     ...    "[start]${SPACE}${SPACE}(Error while starting plugin thyra-plugin-hello-world: plugin is not ready to start.\n). Current plugin status is Up."
```

Comment:
I don't understand what are you waiting for to do the rename in this line

Reply:
need seperate work on the hello world plugin and publish a new release , since the actual PR s already huge one , prefer to merge it, fix the plugins , hello world and node manager , and come back with a small PR with only 2 or 3 changes Hello world and node manager###

Reply:
okay